lib/eventsource.js

```diff
@@ -34,10 +34,10 @@ function EventSource (url, eventSourceInitDict) {
   var self = this
   self.reconnectInterval = 1000
 
-  function onConnectionClosed () {
+  function onConnectionClosed (message=undefined) {
     if (readyState === EventSource.CLOSED) return
     readyState = EventSource.CONNECTING
-    _emit('error', new Event('error'))
+    _emit('error', new Event('error', {message: message}))
```

Comment:
Use object destructure.

Reply:
technically this makes it a breaking change as it breaks compatibility with node 0.12

Reply:
Node.js 0.12 is by far obsolete and unsupported, so I would not worry about it. If it would be a still supported version, then I would agree it would be a breaking change and would accept not including it, but being not the case, it's better to keep up with latests language idioms to have a clean code.

Reply:
I just point it out because it breaks your test suite

Reply:
That's true. Can you be able to update the `.travis.yml` file to remove the deprecated ones?